SolWatch/Game1.cs

```diff
@@ -123,22 +123,39 @@ protected override void Draw(GameTime gameTime)
                 effects: SpriteEffects.None,
                 layerDepth: 0f);
 
-            // Draw Neptune's Orbit
+            foreach (var planetRenderData in SolarSystemData.Data)
+                DrawPlanetOrbit(planetRenderData.Symbol, (int)(planetRenderData.Planet.SemiMajorAxis * kmToPixelsFactor), planetRenderData.Anomaly, planetRenderData.Color);
+
+            spriteBatch.End();
+
+            base.Draw(gameTime);
+        }
+
+        void DrawPlanetOrbit(Texture2D planetSprite, int orbitSize, float angle, Color color)
+        {
             spriteBatch.Draw(
                 texture: orbitTexture,
                 destinationRectangle: new Rectangle(
                     location: screenCenter,
-                    size: maxOrbitSizeInPixels),
+                    size: new(orbitSize * 2, orbitSize * 2)),
```

Comment:
What do you think of using orbitRadius as the variable name instead of orbitSize? That would help make it more clear why it's being multiplied by 2, at least to me.